js/src/models/model.project.js

```diff
@@ -348,6 +348,15 @@ Wu.Model.Project = Wu.Model.extend({
 			title : "Could not update project", 
 			description : result.error
 		});
+
+		if(result.updated[0] == "slug") return;
+
+		// store on server
+		this.store.name = result.project.name;
+
+		// update slug name
+		!window.testMode && this.setSlug(result.project.name);
```

Comment:
`slug` and `name` are two different things.. What's going on here? Is `slug` returned as a `name` key?